bigbluebutton-html5/imports/ui/components/screenshare/component.jsx

```
@@ -491,7 +497,7 @@ class ScreenshareComponent extends React.Component {
           break;
         case PluginSdk.ScreenshareHelperItemPosition.BOTTOM_RIGHT:
           result.bottomRightPluginItems.push(item);
-          break;
+          break;ivan
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will break the flow of the code beforehand


(lint/correctness/noUnreachable)